met-web/src/routes/UnauthenticatedRoutes.tsx

```diff
@@ -11,7 +11,7 @@ const UnauthenticatedRoutes = () => {
     return (
         <Routes>
             <Route path="/" element={<HomePage />} />
-            <Route path="/engagement/view/:engagementId" element={<EngagementView />} />
+            <Route path="/engagement/:engagementId/view" element={<EngagementView />} />
```

Comment:
here you abosulitely have to put the ':' to denote for example the engagementId isn't a path it's a variable name